### ISSUE TITLE
[CHAOSS WEB]: Homepage - Nav Bar

### ISSUE THREAD

**Issue by yigakpoa** [chaoss/Accessibility#12]
### Please state the WCAG Guideline this issue falls under.

Operable

### More on Guidelines

[2.4.3: Focus Order](https://www.w3.org/WAI/WCAG21/quickref/#focus-order) The focus order of elements on the page should be in a logical order

### Please state the WCAG Level this issue falls under.

A

### Description

There is more than one way to locate a specific page in a website.

### Location

Homepage - Nav Bar

### Actual Result

Navigation bar is read as links instead of buttons

### Who is the best person to correct this?

Developer

### Who conducted this A11y test?

Victoria Ottah
